examples/CSharp/Tutorial 1.1 - Hello Window/Program.cs

```diff
@@ -10,10 +10,10 @@ class Program
     {
         private static IWindow window;
 
-        private static void Main(string[] args)
+        private static void Main()
         {
             //Create a window.
-            var options = WindowOptions.Default;
+            WindowOptions options = WindowOptions.Default;
```

Comment:
Using var is a stylistic choice throughout the codebase and it's something we'd like to keep around.